src/Microsoft.Health.Fhir.Azure/ExportDestinationClient/Registration/FhirServerBuilderExportDestinationClientRegistrationExtensions.cs

```diff
@@ -0,0 +1,33 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using EnsureThat;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Health.Extensions.DependencyInjection;
+using Microsoft.Health.Fhir.Azure.ExportDestinationClient;
+using Microsoft.Health.Fhir.Core.Features.Operations.Export.ExportDestinationClient;
+using Microsoft.Health.Fhir.Core.Registration;
+
+namespace Microsoft.Extensions.DependencyInjection
+{
+    public static class FhirServerBuilderExportDestinationClientRegistrationExtensions
```

Comment:
This file and `FhirServerBuilderSecretStoreRgistrationExtensions` could collapse into one class (you can still provide 2 different methods but you don't need 2 classes).